pkgs/dmd/binary.nix

```diff
@@ -37,7 +37,10 @@ in
       lib.optional hostPlatform.isLinux autoPatchelfHook
       ++ lib.optional hostPlatform.isDarwin fixDarwinDylibNames;
 
-    propagatedBuildInputs = [curl tzdata] ++ lib.optional hostPlatform.isLinux glibc;
+    propagatedBuildInputs = [curl tzdata] ++ lib.optional hostPlatform.isLinux [
+      glibc
+      stdenv.cc.cc.libgcc
```

Comment:
I would prefer if we use this commit:
https://github.com/PetarKirov/dlang-nix/commit/0736584ceab34e1c3876a6e9b69e30fe750dc121 where I tried to provide more context about why this change is necessary.